### ISSUE TITLE
Block specific Text direction

### ISSUE THREAD

**Issue by bilel** [codex-team/editor.js#2638]
The actual direction parameter is cool ! But it's applied to all blocks :
```
const editor = new EditorJS({
	   i18n: {
		 direction: 'rtl',
	  },
```
We could add mixed languages into the same document.
I suggest adding a block specific direction that takes the priority over the global document direction.

**Comment by adueck** [comment]
Yes I came here to look for this as well, this would be fantastic, and it is the default way of operating for many text interfaces. 

**Along with `rtl` and `ltr`, it would be great if there were an `auto` direction.**

Notice how this is handled when setting `<textarea dir="auto">`. For example, here's how normal HTML elements respond to the various `dir` attributes.

`<textarea dir="ltr">` - everything goes left-to-right

<img width="354" height="179" alt="Image" src="https://github.com/user-attachments/assets/5663311e-d13b-49fa-8446-e5eaf496d3c7" />

`<textarea dir="rtl">` - everthying goes right-to-left

<img width="354" height="177" alt="Image" src="https://github.com/user-attachments/assets/c15bfc5e-b88f-4b42-977b-420a03627280" />

`<textarea dir="auto">` - direction chosen **based on what the block/paragraph starts with**

<img width="354" height="177" alt="Image" src="https://github.com/user-attachments/assets/b64c24dc-c977-4425-bc09-535e05d6484f" />

This would be great to have with editor.js, as well. When set to `auto` the direction for each block could be detected by checking if the first character is a RTL character or not.
